frontend/src/__tests__/cypress/cypress/pages/projects.ts

```diff
@@ -95,6 +95,26 @@ class ProjectListPage {
   findCreateWorkbenchButton() {
     return cy.findByRole('button', { name: 'Create a workbench' });
   }
+
+  findProjectsTypeDropdown() {
+    return cy.findByLabelText('Options menu');
+  }
+
+  findProjectsTypeDropdownByText(projectType: string) {
+    return cy.contains('button', projectType);
+  }
```

Comment:
Remove unused code.

Reply:
Fixed in [3bafb74](https://github.com/opendatahub-io/odh-dashboard/pull/3028/commits/3bafb7494eb97e74a5c537657c3242a1c73a196e)